src/convertToObject.js

```diff
@@ -11,8 +11,23 @@
  *
  * @return {object}
  */
+
 function convertToObject(sourceString) {
-  // write your code here
+  const stringToObject = {};
+
+  const properties = sourceString
+    .split(';')
+    .map(elem => elem.trim())
+    .filter(elem => elem !== '');
+
+  const propertiesWithoutSpaces = properties
+    .map(elem => elem.split(':').map(item => item.trim()));
+
+  for (const item of propertiesWithoutSpaces) {
```

Comment:
 Можеш з `item`-а деструктурувати ключ і значення 
```suggestion
  for (const item of propertiesWithoutSpaces) {
```